src/main/content/_assets/js/instance-common.js

```diff
@@ -31,6 +31,16 @@ $(document).ready(function(){
             .then(loadAllInfo);
 
     });
+
+    $(".codeready-toggle").on("click", ".bx--toggle__switch", () => {
+        $("#codeready-toggle-value").attr("checked") ? $("#codeready-toggle-value").removeAttr("checked") : $("#codeready-toggle-value").attr("checked", true);
+    });
+
+    $("#modal-edit-instance").on("click", "#modal-edit-instance-save-btn", () => {
+        let instanceName = getActiveInstanceName();
+        fetchAnInstance(instanceName)
+            .then(updateInstance);
+    });
```

Comment:
maybe create a reloadCurrentInstance() method that could be reused by others so then this would be `$("#modal-edit-instance").on("click", "#modal-edit-instance-save-btn", reloadCurrentInstance)`

see if anywhere else does something similar and reuse `reloadCurrentInstance()`

Reply:
oh never mind I thought this was about loading the page, but looks like you are just getting the latest data before updating, so this is fine the way it is.